SUI/SUI/ActionSheetView.swift

```diff
@@ -0,0 +1,184 @@
+// ActionSheetView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Выезжающая снизу вью
+struct ActionSheetView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let dotOffsetNumber: CGFloat = 85
+        static let sliderLightOffsetNumber: CGFloat = -55
+        static let titleText = "A/C is ON"
+        static let descriptionText = "Tap to turn off or swipe up for a fast setup"
+        static let emptyString = ""
+        static let celsusString = "º"
+        static let carDoorImageName = "carDoor"
+        static let onString = "On"
+        static let ventString = "Vent"
+        static let backImageName = "backChevron"
+        static let nextImageName = "nextChevron"
+        static let powerImageName = "power"
+    }
+
+    // MARK: - Public properties
+
+    var body: some View {
+        VStack {
+            Capsule()
+                .fill(.black)
+                .frame(width: 80, height: 3)
+                .padding(.top)
+            HStack(spacing: 60) {
+                titleTextView
+                onButtonView
+            }
+            .padding(.bottom, 20)
+            HStack(spacing: 30) {
+                ColorPicker(Constants.emptyString, selection: $climateViewModel.selectedColor)
+                Spacer()
+                decreaseButtonView
+                celsusTextView
+                increaseButtonView
+                Spacer()
+                Image(Constants.carDoorImageName)
+            }
+            .padding(.horizontal, 40)
+            .padding(.bottom, 20)
+            bottomTextView
+            Spacer()
+        }
+        .frame(width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height / 2)
+        .background(
+            .ultraThinMaterial, in:
+            RoundedRectangle(cornerRadius: 40)
+        )
+        .ignoresSafeArea(.all, edges: .bottom)
+        .offset(y: UIScreen.main.bounds.height / 2 + 120)
+        .offset(y: currentMenuOffsetY)
+        .gesture(dragGesture)
+    }
+
+    // MARK: - Private properties
+
+    @EnvironmentObject private var climateViewModel: ClimateViewModel
+
+    @GestureState private var gestureOffset = CGSize.zero
+
+    @State private var currentMenuOffsetY: CGFloat = 0.0
+    @State private var lastMenuOffsetY: CGFloat = 0.0
+
+    private var titleTextView: some View {
+        VStack(alignment: .leading, spacing: 5) {
+            Text(Constants.titleText)
+                .font(.system(size: 20, weight: .bold, design: .default))
```

Comment:
```suggestion
                .font(.system(size: 20, weight: .bold, design: .default))
```
Используя .system(size: 20, weight: .bold, design: .default) жестко задаешь параметры шрифта, если в .font использовать значения .title, .largeTitle, то приложение может автоматически менять шрифт в зависимости от настроек iOs устройства. Это так, для общего развития, не замечание